views/segments.html

```diff
@@ -173,6 +173,14 @@
     'wooNumberOfOrdersInTheLast': _x('in the last', 'Appears together with wooNumberOfOrdersDays when creating a new WooCommerce segment based on the number of orders.'),
     'wooPurchasedCategory': __('purchased in this category'),
     'wooPurchasedProduct': __('purchased this product'),
+    'wooTotalSpent': __('total spent'),
+    'wooTotalSpentMoreThan': __('more than'),
```

Comment:
Do we need this here? we already have __('more than') a couple of lines above. Also, other translations are here already that could be re-used.

Reply:
I thought about this, and I wouldn't say I like that the key will contain a name from the other type. So, should I use the key with the original name, for example, `wooNumberOfOrdersMoreThan`?

Reply:
I'd rename the key. Maybe just `moreThan`?

Reply:
Yeah, that sounds fair. I wasn't sure if I can rename some existing keys.